src/main/java/io/debezium/connector/yugabytedb/YugabyteDBConnectorConfig.java

```diff
@@ -700,13 +725,45 @@ public JdbcConfiguration getJdbcConfig() {
     public static final Field PLUGIN_NAME = Field.create("plugin.name")
             .withDisplayName("Plugin")
             .withGroup(Field.createGroupEntry(Field.Group.CONNECTION_ADVANCED_REPLICATION, 0))
-            .withEnum(LogicalDecoder.class, LogicalDecoder.DECODERBUFS)
+            .withEnum(LogicalDecoder.class, LogicalDecoder.YBOUTPUT)
             .withWidth(Width.MEDIUM)
             .withImportance(Importance.MEDIUM)
+            .withDefault("yboutput")
             .withDescription("The name of the Postgres logical decoding plugin installed on the server. " +
-                    "Supported values are '" + LogicalDecoder.DECODERBUFS.getValue()
+                    "Supported values are '" + LogicalDecoder.YBOUTPUT.getValue()
                     + "'. " +
-                    "Defaults to '" + LogicalDecoder.DECODERBUFS.getValue() + "'.");
+                    "Defaults to '" + LogicalDecoder.YBOUTPUT.getValue() + "'.");
+
+    public static final Field SLOT_NAME = Field.create("slot.name")
+            .withDisplayName("Slot")
+            .withType(Type.STRING)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTION_ADVANCED_REPLICATION, 1))
+            .withWidth(Width.MEDIUM)
+            .withImportance(Importance.MEDIUM)
+            .withDefault(ReplicationConnection.Builder.DEFAULT_SLOT_NAME)
+            .withValidation(YugabyteDBConnectorConfig::validateReplicationSlotName)
+            .withDescription("The name of the YSQL logical decoding slot created for streaming changes from a plugin." +
+                    "Defaults to 'debezium");
+    
+    public static final Field DROP_SLOT_ON_STOP = Field.create("slot.drop.on.stop")
+            .withDisplayName("Drop slot on stop")
+            .withType(Type.BOOLEAN)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTION_ADVANCED_REPLICATION, 3))
+            .withDefault(false)
```

Comment:
Create a final variable for the default values at the top and then use it here. That will ensure if we ever change the default value then it gets changed everywhere.